src/main/java/mate/project/dto/UserLoginRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.project.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+
+public record UserLoginRequestDto(
+        @NotEmpty
```

Comment:
Not blank is beter here
```suggestion
        @NotBlank
```